networking/mtu.md

```diff
@@ -47,51 +46,72 @@ The following table lists common MTU sizes for {{site.prodname}} environments. B
 - Set the workload endpoint MTU and the tunnel MTUs to the same value (so all paths have the same MTU)
 
+**eBPF mode**
+
+Implementation of NodePorts uses VXLAN tunnel to hand off packets from one node to another, therefore VXLAN MTU setting 
+is used to set the MTUs of workloads (veths) and should be “physical network MTU size minus 50” (see above).
+
 **MTU for flannel networking**
 
-When using flannel for networking, the MTU for network interfaces should match the MTU of the flannel interface. If using flannel with VXLAN, use the “{{site.prodname}} MTU with VXLAN” column in the table above for common sizes. 
+When using flannel for networking, the MTU for network interfaces should match the MTU of the flannel interface.
+- If using flannel with Wireguard encryption, use the "{{site.prodname}} MTU with Wireguard" column in the table above for common sizes.
+- Otherwise, if using flannel with VXLAN, use the “{{site.prodname}} MTU with VXLAN” column in the table above for common sizes.
+
+#### Configure MTU
+
+> **Note**: The updated MTU used by {{site.prodname}} only applies to new workloads. 
+
+Select the appropriate instructions for configuring the MTU.  This is broken down based on installation:
+-  Manifest based installation (if you are not using the quickstart guide, most non-Openshift installs fall under this 
+   category)
+-  Openshift
+-  Operator, non-Openshift (currently just used in the quickstart guide)
 
-#### Configure MTU for workloads
-  
-When you set the MTU, it applies to new workloads. To apply MTU changes to existing workloads, you must restart calico nodes. Restarting the calico/node pods takes values from the ConfigMap and starts rolling updates for any {{site.prodname}} tunnel network interfaces on the node. 
+##### Manifest
 
-Edit the `calico-config` ConfigMap to set values in FelixConfiguration. For example:
+For manifest based installations (i.e. ones that do not use the operator) edit the `calico-config` ConfigMap. For example:
 
 ```bash
 kubectl patch configmap/calico-config -n kube-system --type merge \
   -p '{"data":{"veth_mtu": "1440"}}'
 ```
 
-#### Configure MTU for overlay networking
+After updating the ConfigMap perform a rolling restart of all calico/node pods.
 
-If you are using IP in IP and/or VXLAN for {{site.prodname}} overlay networking, set the tunnel MTU to match the value that you configured for the veth MTU. 
+##### Openshift
 
-Edit `calico-config ConfigMap` to set the MTU tunnel values in FelixConfiguration. For example: 
+For Openshift, edit the {{site.prodname}} operator Installation resource to set the `mtu` field in the `calicoNetwork` section
+of the `spec`.  For example:
 
-```conf
-# Configure the MTU to use
-veth_mtu: "1440" 
+```bash
+oc patch installation.operator.tigera.io default --type merge -p '{"spec":{"calicoNetwork":{"mtu":1440}}}'
 ```
 
-#### eBPF mode
+##### Operator (non-Openshift)
```

Comment:
Do we need separate openshift / non-openshift instructions for the operator? Can we just have one "Using the operator? (e.g., on openshift)" section?

Reply:
Let me rephrase this and see if it looks OK.